web/src/app/main.component.ts

```diff
@@ -3,7 +3,7 @@ import { Router } from '@angular/router';
 import { SessionService, Session, SessionState, UserState } from './session.service';
 
 @Component({
-  selector: 'app-main',
+  selector: '.app-main',
```

Comment:
Why change the selector? Does the dot have a special meaning or was this unintentional?

Reply:
Without the dot, the component was rendered as `<app-main>...</app-main>`. With the dot, the component is rendered as `<div class="app-main">...</div>`. And (for whatever reason) the first case doesn't really apply the padding that I added. Inspecting with Firefox' Web Developer Tools shows that the padding is applied to the `<app-main>` element, but it doesn't have an effect on it's child elements. Maybe the problem was that I hadn't set the `display` property to `block`. I will give that a try.

Reply:
Indeed, this solved the issue. I also changed the display property of the outer `<app-root>` element to `block`, because having block boxes in an inline box sounds wrong.